Formula/fuse-overlayfs.rb

```diff
@@ -18,9 +18,9 @@ class FuseOverlayfs < Formula
 
   def install
     system "autoreconf", "-fis"
-    system "./configure", "--prefix=/", "--sbindir=/sbin"
+    system "./configure", "--prefix=#{prefix}", "--sbindir=#{sbin}"
```

Comment:
```suggestion
    system "./configure", "--prefix=#{prefix}"
```
Nothing is installed in `../sbin`, and the default `configure` value is already correct anyway:
```bash
$ ./configure --help
[...]
  --prefix=PREFIX         install architecture-independent files in PREFIX
                          [/usr/local]
  --exec-prefix=EPREFIX   install architecture-dependent files in EPREFIX
                          [PREFIX]
[...]
  --sbindir=DIR           system admin executables [EPREFIX/sbin]
```